pkgs/top-level/python-packages.nix

```diff
@@ -126,6 +126,8 @@ let
     mpi = pkgs.openmpi;
   };
 
+  bootstrapped-pip = callPackage ../development/python-modules/bootstrapped-pip { };
```

Comment:
is this actually needed? i see bootstrapped-pip as an implementation detail of ``development/python-modules/generic/``